ASIM/dev/Delete-SentinelFunction/Delete-LASavedFunction.ps1

```diff
@@ -0,0 +1,107 @@
+    <#
+        .SYNOPSIS
+        Deletes saved searches from Log Analytics workspace
+
+        .DESCRIPTION
+        Deletes functions from Log Analytics workspace.
+        Takes a list of strings for the function names.
+
+        .PARAMETER FunctionName
+        Specifies a comma delimited list of saved function name to be deleted. Accepts wildchars. 
```

Comment:
I assume you meant "wildcards". Also, which are supported: the windows * and ? If so. "Accepts the wildcards * and ?.

Reply:
Forget about it, I will edit the texts.

Reply:
accepts both * and ?